components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -135,8 +137,11 @@ export default class SubMenuItem extends React.PureComponent<Props, State> {
     }
 
     public render() {
-        const {id, postId, text, selectedValueText, subMenu, icon, filter, ariaLabel, direction, styleSelectableItem, extraText, renderSelected, rightDecorator} = this.props;
+        const {id, postId, text, selectedValueText, subMenu, icon, filter, ariaLabel, direction, styleSelectableItem, extraText, renderSelected, rightDecorator, showMenu, subMenuClass} = this.props;
```

Comment:
There was a prop called `subMenuClass` but it was unused. Using it here. Not sure why it was unused, this is one of the reasons the CI tests are failing.